metecho/api/models.py

```diff
@@ -731,10 +772,12 @@ def add_reviewer(self, user):
             self.save()
 
     def get_repo_id(self):
-        return self.epic.project.get_repo_id()
+        return self.root_project.get_repo_id()
 
     def get_base(self):
-        return self.epic.branch_name
+        if self.epic:
+            return self.epic.branch_name
+        return self.project.branch_name
```

Comment:
I'm not entirely sure if this is what we want: if a task is attached to a Project and not an Epic, the base branch is the actual project branch

Reply:
I think that sounds right